### ISSUE TITLE
Alternative combination for i-hook

### ISSUE THREAD

**Issue by KrasnayaPloshchad** [adobe-fonts/source-serif#95]
I found a variant of ỉ on the _Dictionarium Annamiticum Lusitanum et Latinum_ (i.e. “Dictionary of Vietnamese, Portuguese and Latin”) that ratains the tittle, in this case the hook is placed at the right of character frame, and the altitude of hook is sightly lower. To implement this variant, you can just modify the mark attachment for 0069 0307 0309.

See: https://commons.wikimedia.org/wiki/File:Alexandre_de_Rhodes_-_Dictionarium_Annamiticum_(1651)_-_c.34_-_cropped_on_b%C3%AD_and_b%E1%BB%89.png

**Comment by frankrolf** [comment]
Thank you. 
So this would basically be an alternate of U+1EC9?
I was about to suggest implementing your suggestion as a default form for the i + combining hook sequence, but it seems to be more reasonable to insist on the combining dot if needed.

**Comment by KrasnayaPloshchad** [comment]
Yes, if you want to implement a variant for U+1EC9, I won’t oppose. BTW I've seen Vietnamese text may retain the tittle in certain cases, some of them can be seen at the Wikimedia Commons.
https://commons.wikimedia.org/wiki/Category:Hard-dotted_I_in_Vietnamese
